core/drivers/imx_ocotp.c

```diff
@@ -9,18 +9,34 @@
 #include <imx.h>
 #include <io.h>
 #include <drivers/imx_ocotp.h>
+#include <kernel/delay.h>
 #include <kernel/delay_arch.h>
 #include <kernel/tee_common_otp.h>
 
 #define OCOTP_CTRL			0x0
+#define OCOTP_CTRL_SET			0x4
 #define OCOTP_CTRL_CLR			0x8
+#define OCOTP_TIMING			0x10
+#define OCOTP_DATA			0x20
 
+#define OCOTP_CTRL_WR_UNLOCK_KEY	0x3E77
+
```

Comment:
Readability of macros is so bad here, can we use only tabs here ?
I am not sure if only tabs are already used here.